warehouse/utils/utils.go

```diff
@@ -765,6 +765,17 @@ func GetConfigValue(key string, warehouse WarehouseT) (val string) {
 	}
 	return val
 }
+func GetConfigValueBoolString(key string, warehouse WarehouseT) string {
+	config := warehouse.Destination.Config
```

Comment:
can we do `return strconv.FormatBool(GetConfigValue(key, warehouse) == "true")`

Reply:
getConfigValue will return empty string if interface is not empty string

Reply:
👍